usaspending_api/references/reference_helpers.py

```diff
@@ -5,6 +5,10 @@
 from usaspending_api.accounts.models import FederalAccount, TreasuryAppropriationAccount
 
 
+def remove_federal_accounts(tas_tuple=None):
+    FederalAccount.objects.filter(treasuryappropriationaccount__isnull=True).distinct().delete()
```

Comment:
this only runs once anyway. (we are removing the bad federal accounts that are currently in db)